### PR TITLE
added .capacity() method in vectors for cpp

### PR DESCRIPTION
**Pull request by pratikkumar399**
### Description

Hi, there.
I have added a capacity() function to track the capacity of the vectors in cpp as it grows dynamically.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by pratikkumar399** [comment]
@SSwiniarski can you please review this pr?

File: content/cpp/concepts/vectors/vectors.md
Comment (SSwiniarski) [inline content/cpp/concepts/vectors/vectors.md:73]:
```suggestion
`.capacity()` is used to retrieve the current capacity of a `std::vector`. The capacity of a vector represents the number of elements it can hold without needing to allocate additional memory. The vector may have more capacity than its current size to reduce the need for frequent reallocations and copying when elements are added. 
```

File: content/cpp/concepts/vectors/vectors.md
Comment (SSwiniarski) [inline content/cpp/concepts/vectors/vectors.md:74]:
```suggestion
Initially, the capacity is likely to be small, but as elements are added using `push_back()`, the capacity will grow to accommodate the added elements. The exact growth behavior may vary depending on the C++ implementation, but it's typically designed to minimize reallocations and copying.
```

File: content/cpp/concepts/vectors/vectors.md
Comment (SSwiniarski) [inline content/cpp/concepts/vectors/vectors.md:92]:
```suggestion
```
remove blank line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@pratikkumar399 I did a first review and left a few comments.

**Comment by pratikkumar399** [comment]
@SSwiniarski done with the requested changes. Can you please review?

**Comment by pratikkumar399** [comment]
@SSwiniarski can you please merge the pr, if it looks good to you.

**Comment by SSwiniarski** [comment]
> @SSwiniarski can you please merge the pr, if it looks good to you.

Another maintainer needs to review this before it is merged. Have patience.

File: content/cpp/concepts/vectors/vectors.md
Comment (caupolicandiaz) [inline content/cpp/concepts/vectors/vectors.md:78]: instead of creating a new vector for this function I would suggest to use and modify the vector "order" used in the other examples within this section

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@pratikkumar399 I've added a couple of comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by pratikkumar399** [comment]
@caupolicandiaz I have implemented your suggestion.

**Review by caupolicandiaz — APPROVED** [review]
@pratikkumar399 @SSwiniarski LGTM

**Comment by pratikkumar399** [comment]
@caupolicandiaz I have rectified the lint errors. Can you please review it?

**Pull request merged**
